apps/server/dev-setup/init-db.js

```diff
@@ -1,7 +1,89 @@
-import createProject from './create-project.js'
+import { allOrganizations } from 'shared/src/utils/iterables.js'
+import setupProjects from './setup-projects.js'
+import setupUsers from './setup-users.js'
+import { createOrganization } from '../src/models/queries/organization-queries.js'
+import { dropTables, synchroniseModels } from '../src/connect.js'
+// import { createUser, getUserById } from '../src/models/queries/user-queries.js'
+// import { projectInitializing, projectArchiving, projectCreated, projectFailed, projectAddUser, getProject, projectRemoveUser } from '../src/models/queries/project-queries.js'
+// import { getEnvironment, environmentInitializing, environmentCreated, environmentFailed } from '../src/models/queries/environment-queries.js'
+// import { setEnvironmentPermission } from '../src/models/queries/permission-queries.js'
+// import { repositoryCreated, repositoryFailed, repositoryInitializing, repositoryDeleting, updateRepository, deleteRepository } from '../src/models/queries/repository-queries.js'
 
 export const initDb = async () => {
-  return Promise.all([
-    createProject(),
-  ])
+  await dropTables()
```

Comment:
Pourquoi un drop des tables? Vous perdriez tous les données à livraison.

Reply:
Sauf si vous supprimez ou vider le fichier init-db.js à chaque redémarrage et livraison

Reply:
La PR est en `work in progress`, on changera ça au moment de la review finale

Reply:
TODO